extern/ProtoGeometry/ProtoGeometry.XML

```diff
@@ -3550,7 +3584,8 @@
             <summary>
             Scale in one dimension by base and 2 pick points.  The scaling axis is defined by the line between base and pick0.
             </summary>
-            <search>resize,size,from,to,scale1d,1d</search>
+            <search>resize,size,from,to,scale1d,1d,lines</search>
+            <weights>0.5,0.5,0.5,0.5,0.5,0.5,0.4</weights>
         </member>
         <member name="M:Autodesk.DesignScript.Geometry.CoordinateSystem.Scale2D(Autodesk.DesignScript.Geometry.Plane,Autodesk.DesignScript.Geometry.Point,Autodesk.DesignScript.Geometry.Point)">
             <summary>
```

Comment:
Did you make this change in the closed source repository, too? We'll need to update it to transfer the weights.